components/Base/CollapsableItem.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <Disclosure v-slot="{ open }">
-    <div
+    <li
```

Comment:
I think this is not the best practice to make single component as a `li`. It is independent and can be used outside of list. I think it's good to wrap the component into `li` in usage when it should be a list item 👌 example in `components/ProductsList.vue`

Reply:
```
<ul>
          <li>
            <BaseCollapsableItem
              v-for="item in listData"
              :key="item.id"
              :info="item"
            />
          </li>
        </ul>
```
is this what you had in mind?

Reply:
almost 👌 now `v-for` should iterate on `li` elements and everything will be good 👍